packages/auth/src/providers/cognito/utils/signInHelpers.ts

```diff
@@ -91,6 +91,7 @@ export async function handleCustomChallenge({
 		ANSWER: challengeResponse,
 	};
 
+	// TODO (ashwinkumar6): should we pass usename here as well ?
 	const deviceMetadata = await tokenOrchestrator?.getDeviceMetadata();
```

Comment:
Open Question
Should we add username here as well ?

Reply:
I think so, otherwise this would be empty (LastAuthUser) should be `undefined` because is during signIn

Reply:
thanks updated, we're passing username here as well